### ISSUE TITLE
ch-4-ex-1

### ISSUE THREAD

**Issue by ltanase77** [oauthinaction/oauth-in-action-code#24]
I cannot obtain the protected resource based on the current setup. 
Following app.get("/fetch resource") router I get back the console.log message: _Making request with access token OzH6eSierpbDo7YaeFcp861YmvHKBTPf_, but I get the error view with a code of 401 
The resource object resulting from the post request to app.post("/resource") router is as follows:
```
Response {
  statusCode: 401,
  headers:
   { 'x-powered-by': 'Express',
     'access-control-allow-origin': '*',
     date: 'Mon, 01 Jul 2019 20:19:43 GMT',
     connection: 'close',
     'content-length': '0' },
  body: <Buffer >,
  url: undefined }
```
It seems that the nosql.one function does not find the token in the database (as the incoming token message shows the value of the token) although if i inspect the nosql file the token is written on the file.
The docs for nosql seems very thin, i can find any help in them for solving the problem.

**Comment by Babagilo** [comment]
If I run the protectedResource.js from Apple shell, I am able to read the doc out; however, if I execute it from command line of VSC, then I encounter the same "missing record" system.

The content of the book is very insightful, it's a shame to see the code is very poorly maintained.

**Comment by jricher** [comment]
Should be fixed by #44
